pangocffi/color.py

```diff
@@ -107,28 +82,26 @@ def parse_color(self, spec: str) -> bool:
 
     def to_string(self):
         """
-        Returns a textual specification of color in the
-        hexadecimal form #rrrrggggbbbb, where r, g and b
-        are hex digits representing the red, green, and
-        blue components respectively.
+        Returns a string representation of the color in the hexadecimal
+        format `#RRRRGGGGBBBB`.
 
         :return:
-            string hexadecimal
+            the color string
         """
         string = ffi.gc(
             pango.pango_color_to_string(self._pointer),
             glib.g_free,
         )
         return ffi.string(string).decode("utf-8")
 
-    def __eq__(self, col: "Color") -> bool:
-        if isinstance(col, Color):
+    # TODO: remove this and go back to using _COPY_METHOD
+    def __copy__(self) -> "Color":
+        return Color(self.red, self.green, self.blue)
```

Comment:
To get the tests working, we also need to override the `copy()` method itself.

I've also updated the comment to point to a new issue I created here: https://github.com/leifgehrmann/pangocffi/issues/49

```suggestion
    # TODO: remove this and go back to using _COPY_METHOD
    # See https://github.com/leifgehrmann/pangocffi/issues/49
    def copy(self) -> "Color":
        return self.__copy__()

    def __copy__(self) -> "Color":
        return Color(self.red, self.green, self.blue)
```